phpstan.neon

```diff
@@ -1,6 +1,10 @@
 parameters:
     tmpDir: var/cache/ecs
     level: 5
+
+    paths:
+        - src
+
```

Comment:
It's best practise to have paths in configs, because it's now impossible to forget them.